### DISCUSSION TITLE
Can I get the index of the sampled node in the original data after using NeighborLoader for sampling

### DISCUSSION THREAD

**Discussion by Danieltobesixabs** [pyg-team/pytorch_geometric#4466] [Q&A] [answered]
Hello，I've been reproducing a piece of paper recently，when i use the NeighborLoader to sample, I need to get the index of the sampled node in the original data after using NeighborLoader for sampling, but i read the documentation, i found that i can`t get it. So could you tell me how can i get the index？

**Comment by rusty1s** [comment, marked as answer]
You can add the information to the original data:
```python
data.n_id = torch.arange(data.num_nodes)
```

**Reply by Danieltobesixabs** [reply]
Thank you for help, i solved this problem!
